.drone.star

```diff
@@ -331,8 +336,14 @@ config = {
 				'webUISharingFolderPermissionsGroups': 'OCISSharingFolderPermissionsGroups',
 				'webUISharingFolderAdvancedPermissionsGroups': 'OCISSharingFolderAdvPermissionsGrp',
 				'webUISharingPermissionToRoot': 'OCISSharingPermissionToRoot',
-				'webUIResharing': 'OCISResharing', # for now run this suite by itself see https://github.com/owncloud/ocis/issues/736
-				'webUIResharingToRoot': 'OCISResharingToRoot',
+				# for now run this suite by itself see https://github.com/owncloud/ocis/issues/736
+				'OCISResharing1': [
+					'webUIResharing1'
+				],
+				'OCISResharing2': [
+					'webUIResharing2',
+					'webUIResharingToRoot'
```

Comment:
I dont think we need to run `ToRoot` suites in ocis

Reply:
I was also thinking the same but it was already there, so I just included it.